be/src/vec/exec/vjdbc_connector.cpp

```diff
@@ -83,8 +83,12 @@ Status JdbcConnector::close(Status /*unused*/) {
     if (_is_in_transaction) {
         RETURN_IF_ERROR(abort_trans());
     }
-    JNIEnv* env;
-    RETURN_IF_ERROR(JniUtil::GetJNIEnv(&env));
+    JNIEnv* env = nullptr;
+    Status status = JniUtil::GetJNIEnv(&env);
```

Comment:
There are many APIs that call GetJNIEnv, and use RETURN_IF_ERROR(JniUtil::GetJNIEnv(&env));
I think we should add check in GetJNIEnv if the result == nullptr, should return an error status.